app/admin/reports/realtime/bad_routings.rb

```diff
@@ -29,7 +29,7 @@
   controller do
     def scoped_collection
       length = params[:q][:time_interval_eq].to_i
```

Comment:
remove this unused variable

Reply:
only variable?

Reply:
what do you mean ?

Reply:
or all line with params

Reply:
yes, whole line
the method should looks like
```
def scoped_collection
  super.detailed_scope
end
```